README.md

```diff
@@ -257,6 +259,16 @@ Notes:
 
 ### Special classes within teacher content
 
+####Teacher Content with NGSS tag
+
+```html
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```

Comment:
The `<span class="ost-tag-ngss">` should probably exclude "Students who demonstrate understanding can: " since I'm guessing it'll be repeated for every tag, no?